packages/esm-patient-chart-app/src/index.ts

```diff
@@ -32,12 +32,17 @@ function setupOpenMRS() {
   registerBreadcrumbs([
     {
       path: spaBasePath,
-      title: 'Patient',
+      // @ts-ignore
```

Comment:
Well, I don't like the `// @ts-ignore`. I'd prefer the `Promise.resolve()` to that.

Reply:
Sorry for the same.
Thanks!